vignettes/custom_appearance.Rmd

```diff
@@ -90,3 +94,239 @@ lyt3 <- basic_table() %>%
+  prov_footer = "file: /path/to/stuff/that/lives/there HASH:1ac41b242a"
+) %>%
+  split_cols_by("ARM") %>%
+  split_rows_by("SEX", page_by = TRUE, page_prefix = "Patient Subset - Gender", split_fun = drop_split_levels) %>%
+  split_rows_by("STRATA1") %>%
+  analyze("AGE", mean, format = "xx.x")
+
+# without inset
+tbl <- build_table(lyt, DM)
+tbl
+
+
+# with inset
+table_inset(tbl) <- 5
+tbl
+```
+
+### Horizontal Separation
+
+A character value can be specified to modify the horizontal separation between column headers and the table. Horizontal separation applies when:\
+1) separating title + subtitles from the column labels + topleft materials,\
+2) column labels + top left material from row labels + cells,\
+3) row labels + cells from footer content, and\
+4) Referential footnotes from main + provenance content \emph{only if} there would be something on both sides of the divider.\
+
+Below, we replace the default line with "=".
+
+```{r}
+tbl <- basic_table() %>%
+  split_cols_by("Species") %>%
+  add_colcounts() %>%
+  analyze(c("Sepal.Length", "Petal.Width"), function(x) {
+    in_rows(
+      mean_sd = c(mean(x), sd(x)),
+      var = var(x),
+      min_max = range(x),
+      .formats = c("xx.xx (xx.xx)", "xx.xxx", "xx.x - xx.x"),
+      .labels = c("Mean (sd)", "Variance", "Min - Max")
+    )
+  }) %>%
+  build_table(iris, hsep = "=")
+tbl
+```
+
+
+### Section Dividers
+
+A character value can be specified as a section divider which succeed every group defined by a split instruction. Note, a trailing divider at the end of the table is never printed.
+
+Below, a "+" is repeated and used as a section divider.
+
+```{r}
+lyt <- basic_table() %>%
+  split_cols_by("Species") %>%
+  analyze(head(names(iris), -1), afun = function(x) {
+    list(
+      "mean / sd" = rcell(c(mean(x), sd(x)), format = "xx.xx (xx.xx)"),
+      "range" = rcell(diff(range(x)), format = "xx.xx")
+    )
+  }, section_div = "+")
+
+build_table(lyt, iris)
+```
+
+Section dividers can be set to " " to create a blank line.
+
+```{r}
+lyt <- basic_table() %>%
+  split_cols_by("Species") %>%
+  analyze(head(names(iris), -1), afun = function(x) {
+    list(
+      "mean / sd" = rcell(c(mean(x), sd(x)), format = "xx.xx (xx.xx)"),
+      "range" = rcell(diff(range(x)), format = "xx.xx")
+    )
+  }, section_div = " ")
+
+build_table(lyt, iris)
+```
+
+Separation characters can be specified for different row splits. However, only one will be printed if they "pile up" next to each other. 
+
+```{r}
+lyt <- basic_table() %>%
+  split_cols_by("ARM") %>%
+  split_rows_by("RACE", section_div = "=") %>%
+  split_rows_by("STRATA1", section_div = "~") %>%
+  analyze("AGE", mean, var_labels = "Age", format = "xx.xx")
+
+build_table(lyt, DM)
+```
+
+### Indent Modifier
+
+Tables by default do not have an indent at the split level. A custom indent value can be supplied with the `indent_mod` argument within a split function.
```

Comment:
I'm not sure what you're tryign to say here, but what it *seems* to say is incorrect.

tables  (or rather the table printing method) does have indenting by default at each level of splitting. `indent_mod` modifies that (to 0 via negative values if desired) default amount of indenting.

Please revise this text to be clearer (and correct).